packages/web-app-webfinger/src/views/Resolve.vue

```diff
@@ -62,7 +62,7 @@ export default defineComponent({
       } catch (e) {
         console.error(e)
         if (e.response?.status === 401) {
-          return authService.handleAuthError(unref(route))
+          return authService.handleAuthError(unref(route), { forceLogout: true })
```

Comment:
This change here is not on master. I'm not sure if we should add it, but it might be the solution to the webfinger issues. AFAICT this now should behave like it was pre v8.0.2.